cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		t.Logf("Could not return a pipe that will be connected to the command's standard output, received err: %v", err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		t.Log(err)
```

Comment:
Again, I'm not sure you can continue after this. use t.Fatal

Reply:
t.Fatalf("cmd.Start: got %v, want nil", err)

Reply:
Fixed it